near-sdk/src/types/mod.rs

```diff
@@ -13,6 +13,9 @@ pub use self::account_id::{AccountId, ParseAccountIdError};
 mod gas;
 pub use self::gas::Gas;
 
+#[cfg(test)]
+mod event_tests;
```

Comment:
Is it actually the right place for the tests? Should we move it to `near-sdk/tests`?